package.json

```diff
@@ -17,24 +29,30 @@
     "colorette": "^2.0.20",
     "electron-is-dev": "^2.0.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1"
+    "react-dom": "^18.2.0"
   },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "dev": "concurrently -k \"BROWSER=none yarn start\" \"yarn:electron\"",
-    "electron": "electron .",
-    "build-electron": "yarn build && electron-builder --linux --windows",
-    "build-electron-linux": "yarn build && electron-builder --linux",
-    "build-electron-win": "yarn build && electron-builder --windows"
+  "devDependencies": {
+    "@types/react": "^18.2.14",
+    "@types/react-dom": "^18.2.6",
+    "@typescript-eslint/eslint-plugin": "^5.61.0",
+    "@typescript-eslint/parser": "^5.61.0",
+    "@vitejs/plugin-react-swc": "^3.3.2",
+    "concurrently": "^8.2.0",
+    "electron": "^25.2.0",
+    "electron-builder": "^24.4.0",
+    "eslint": "^8.44.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-react-refresh": "^0.4.1",
+    "tsc-watch": "^6.0.4",
```

Comment:
Why do you have ts-watch and vite? Doesn't vite have dev server 🤔

Reply:
tsc-watch is for electron file and vite is for website